app/controllers/api/v1/users_controller.rb

```diff
@@ -95,4 +94,8 @@ def validate_password(user, password)
         render json: {status: 500, message: 'Invalid Username/Password'}
       end
     end
+
+    def initialize_user
```

Comment:
Either method name should be user or you should not memorize it in user variable. direct use that method,
Or if you are not using this method any other place then revert the code.